projects.md

```diff
@@ -13,9 +13,8 @@ permalink: /projects/
 ## Appstore replacement
 
 * [The Rebble Store](https://github.com/pebble-dev/rebble-store) *(formerly The Panic Store)*
```

Comment:
If you're replacing *Panic* then this source link should be updated with the new source repo too - though a footnote about the prior effort would be nice, similar to *Android*?

Reply:
Ah, I didn't know there was a new repo for that! I'll fix that ASAP.

Reply:
I just realized I can't seem to find the new repo. Is there a link I can find that at, or the name of the repo?